### PR TITLE
Add toleration values to newrelic infra chart

### PR DESCRIPTION
**Pull request by diclophis**
This PR adds custom `tolerations` configuration for the newrelic-infrastructure chart. In this way users of the chart may decide to schedule the daemonset on master / other nodes as need-be.

### CODE REVIEW

**Comment by diclophis** [comment]
@mattfarina is there anything else we need to do to help this PR get through the process?

File: stable/newrelic-infrastructure/templates/daemonset.yaml [hunk 1]
```diff
@@ -84,4 +84,8 @@ spec:
             - key: newrelic-infra.yml
               path: newrelic-infra.yml
         {{- end }}
+      {{- if .Values.daemonset.tolerations }}
+      tolerations:
+{{ toYaml .Values.daemonset.tolerations | indent 8 }}
```
Comment (unguiculus) [inline stable/newrelic-infrastructure/templates/daemonset.yaml]: Just call it `tolerations`, please. This would be consistent with other charts.

**Pull request merged**